messaging-client-databus/src/main/java/com/inmobi/messaging/consumer/databus/DatabusConsumer.java

```diff
@@ -125,6 +126,29 @@ protected void initializeConfig(ClientConfig config) throws IOException {
     } else {
       parseClusterNamesAndMigrateCheckpoint(config, rootDirSplits);
     }
+    /*
+    This timer task starts after an initial delay of 30 minutes
+    and runs every 5 minutes. This guy is responsible for identifying new
+    collector output files and initialising partition readers for them
+     */
+    int initialDelay = config.getInteger(initialDelayForDiscoverer,
```

Comment:
Move all this logic to different method and invoke only if stream type is COLLECTOR.